<?xml version='1.0' encoding='utf-8'?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Поиск файлов</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout">
    <item>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QLineEdit" name="directoryEdit" />
      </item>
      <item>
       <widget class="QPushButton" name="browseButton">
        <property name="text">
         <string>Обзор...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelExtension">
        <property name="text">
         <string>Расширение:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="extensionEdit" />
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelContent">
        <property name="text">
         <string>Содержимое:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="contentEdit" />
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelSize">
        <property name="text">
         <string>Размер (KB):</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout">
        <item>
         <widget class="QSpinBox" name="minSizeSpin" />
        </item>
        <item>
         <widget class="QLabel" name="labelTo">
          <property name="text">
           <string>до</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="maxSizeSpin">
          <property name="maximum">
           <number>1024</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="resultList" />
    </item>
    <item><widget class="QLabel" name="label_regex"><property name="text"><string>Регулярное выражение:</string></property></widget></item><item><widget class="QLineEdit" name="lineEdit_regex" /></item><item><widget class="QLabel" name="label_date_from"><property name="text"><string>Дата с:</string></property></widget></item><item><widget class="QDateEdit" name="dateEdit_from" /></item><item><widget class="QLabel" name="label_date_to"><property name="text"><string>Дата по:</string></property></widget></item><item><widget class="QDateEdit" name="dateEdit_to" /></item><item>
     <widget class="QProgressBar" name="progressBar" />
    </item>
    <item>
     <widget class="QLabel" name="statusLabel">
      <property name="text">
       <string>Готово</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QPushButton" name="searchButton">
        <property name="text">
         <string>Начать поиск</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="text">
         <string>Очистить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources />
 <connections />
</ui>


